docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1780 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def murlin(T, min_m=3, max_m=3, max_k=1, min_dist=None, decay_init_value=None, decay=None):
</pre><p>Since we have a public function called <code>motifs()</code>, I wondering if it is less confusing and more obvious to name this function <code>discords()</code> instead of <code>murlin()</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='69'/>

Reply:
Yes...that sounds good! I will change the name of its private function as well. (so.... `_discords()`)